tests/extmod/tls_dtls.py

```diff
@@ -0,0 +1,55 @@
+# Test DTLS functionality including timeout handling
+
+try:
+    from tls import PROTOCOL_DTLS_CLIENT, PROTOCOL_DTLS_SERVER, SSLContext, CERT_NONE
+    import socket, io
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+
+class DummySocket(io.IOBase):
+    def __init__(self):
+        self.write_buffer = bytearray()
+        self.read_buffer = bytearray()
+
+    def write(self, data):
+        self.write_buffer.extend(data)
+        return len(data)
```

Comment:
The `write_buffer` is never actually used.  So this function could just do `return len(data)`.

Reply:
Changed!